packages/varlet-cli/src/shared/ejsRender.ts

```diff
@@ -0,0 +1,52 @@
+import ejs from 'ejs'
+import fs from 'fs-extra'
+import path from 'node:path'
+import prettier from 'prettier'
+
+export async function ejsRender(filePath: string, options: any): Promise<void> {
+  try {
+    let prettierCode = ''
+    // 根目录template 绝对路径
+    const templatePath = path.resolve(__dirname, `../../template/create`)
+    // 获取当前渲染文件的 各种 参数 such as ext
+    const file = path.parse(filePath)
+    // 编译根目录 创建的根目录
+    const dest = path.resolve(`${process.cwd()}/src`, options.projectName)
+    // 当前 需要编译的 ejs文件
+    const readFilePath = path.resolve(dest, file.dir, `${file.name}.ejs`)
+    // 转换 之后的 js or ts or vue 文件
+    const outputFilePath = path.resolve(dest, filePath)
+    // 是一个buffer
+    const templateCode = await fs.readFile(readFilePath)
+    // 编译当前code
+    const code = ejs.render(templateCode.toString(), options)
+    // 获取后缀
+    const extname = path.extname(filePath).replace(/[.]/g, '')
+    const opts = await prettier.resolveConfig(templatePath)
+    try {
+      switch (extname) {
+        case 'ts':
+          prettierCode = prettier.format(code, { parser: 'babel', ...opts })
+          break
+        case 'tsx':
+          prettierCode = prettier.format(code, { parser: 'babel', ...opts })
+          break
+        case 'jsx':
+          prettierCode = prettier.format(code, { parser: 'babel', ...opts })
+          break
+        case 'js':
+          prettierCode = prettier.format(code, { parser: 'babel', ...opts })
+          break
+        default:
+          prettierCode = prettier.format(code, { parser: extname })
+          break
+      }
+    } catch (err) {
+      console.log(err)
+    }
```

Comment:
prettier is unnecessary

Reply:
I think it can be used here. But here are a lot of duplicate code. I think can write a inline function insert into. then remove try catch block. Because the wrapper already has try catch block.

Reply:
Make sure the code style of ejs template is correct and prettier can be removed

Reply:
Right, But the format func looks more like an insurance : ) (If removed it. We must sure the ejs template syntax is right.

Reply:
The template code is minimal and does not change ofter, so I think we can make sure this

Reply:
okay. Will wait to be resolved